src/input_parsers/AcasNeuralNetwork.cpp

```diff
@@ -76,8 +76,8 @@ void AcasNeuralNetwork::evaluate( const Vector<double> &inputs, Vector<double> &
     double *input( inputVector.data() );
     double *output( outputVector.data() );
 
-    memset( input, 0, num_inputs( _network ) );
-    memset( output, 0, num_outputs( _network ) );
+    std::fill_n( input, num_inputs( _network ), 0 );
+    std::fill_n( output, num_outputs( _network ), 0 );
```

Comment:
This actually creates a seg fault (for both Linux and Windows).

The argument outputs is an empty vector, which we append to on line 96. So outputs.size() is 0, which means the double array output is not the right size. So when fill_n tries to fill the array, it fills more than is allocated.

The simplest fix might be to use num_outputs( _network ) instead of outputs.size(). That is working for me now